backend/drone.js

```diff
@@ -0,0 +1,60 @@
+const dgram = require("dgram");
+const wait = require("waait");
+const app = require("express")();
+const http = require("http").Server(app);
+const io = require("socket.io")(http);
+const throttle = require("lodash/throttle");
+const commandDelays = require("./commandDelays");
+
+const PORT = 8889;
+const HOST = "192.168.10.1";
+const drone = dgram.createSocket("udp4");
+drone.bind(PORT);
+
+function parseState(state) {
+  return state.split(":").map((x) => x.split(":"));
+}
+
+const droneState = dgram.createSocket("udp4");
+droneState.bind(8890);
```

Comment:
this port is different from the `PORT`above. do you need both or can one be cleaned up maybe?

Reply:
I need this port to send information for `droneState`
![state port](https://user-images.githubusercontent.com/58105956/109983021-bbda8900-7d02-11eb-9d91-4119534efe7a.jpg)

Reply:
oh, I see - drone state needs a different port than where we're sending the commands to. gotcha!